### PR TITLE
Pattern

### PR DESCRIPTION
**Pull request by SumitKanth**
Added all pattern

### CODE REVIEW

**Comment by aaryahjolia** [comment]
Please write markdown files properly, The content is not visible properly. 
[https://github.com/aaryahjolia/dsa_competitive-coding/blob/191f8e5a9184c77119a72e76d94b5173971b4421/Pattern/Pattern1-5.md](https://github.com/aaryahjolia/dsa_competitive-coding/blob/191f8e5a9184c77119a72e76d94b5173971b4421/Pattern/Pattern1-5.md).
Also make a seperate .md file for each pattern.

**Comment by SumitKanth** [comment]
1. Please write markdown files properly, The content is not visible properly. 
-> Can you give example of how to write down markdown files, The content which is not visible is the shape of the pattern which we will get in output.

2. Also make a seperate .md file for each pattern.
-> I have 28 patterns, so I have to have 28 files ?

**Comment by aaryahjolia** [comment]
You can include shape inside 3 backtick (`) Like this : ```Hello inside backticks```. 
See other markdown files like that of sorting files for reference.

It's advisable to make seperate files as it will look good to practice one question at a time but you can do only 5 files also but with proper marking in .md files.

**Comment by SumitKanth** [comment]
Changes done , you can check it and merge it

**Comment by SumitKanth** [comment]
I have contributed under HCSOC22'

**Comment by SumitKanth** [comment]
plz watch the changes
I have contributed under HSOC22

File: Pattern/Pattern1.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        cout << endl;
+        row++;
+    }
+    return 0;
+}
+```
+
+## Output
+```
+n = 3;
```
Comment (aaryahjolia) [inline Pattern/Pattern1.md:36]: Don't write n=3 in output, Instead write it in seperate line before ```

File: Pattern/Pattern1.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+*
+* *
+* * *
+```
+
+## Prerequisite
```
Comment (aaryahjolia) [inline Pattern/Pattern1.md:42]: Instead of prerequisite section, Add the url to gfg ide.

**Review by aaryahjolia — COMMENTED** [review]
Add GFG url to every md file's code instead of prerequisite section.
In pattern 4 and 5, Write seperate code and output for each pattern.
Write input like n=3 before ```, Don't write in output frame.

**Comment by SumitKanth** [comment]
How to add GFG url ?

**Comment by aaryahjolia** [comment]
Go to gfg ide,Paste your code and click on Run+Generate URL,Paste that url back to your .md file

**Comment by SumitKanth** [comment]
Changes done you can check and merge

**Comment by aaryahjolia** [comment]
LGTM!

**Pull request merged**